test/pathOrLazy.js

```diff
@@ -0,0 +1,53 @@
+import sinon from 'sinon';
+
+import * as RA from '../src';
+import eq from './shared/eq';
+
+describe('pathOrLazy', function() {
+  const obj = {
+    a: { b: { c: 1, d: ['x', 'y', 'z'] } },
+    e: NaN,
+    f: null,
+    g: undefined,
+  };
+
+  it('returns value for requested path', function() {
```

Comment:
We're using more BDD approach of describing our tests. Every test should start with `should`. In this case `should return value for requested path`.